Tests on master are broken

Since early hours of Jul 12.

```
Step 5/9 : RUN cd packages/proto-loader &&   npm install @types/mocha &&   npm install --unsafe-perm
 ---> Running in 714117e8e0ba
npm notice created a lockfile as package-lock.json. You should commit this file.
+ @types/mocha@8.0.0
added 1 package from 6 contributors and audited 1 package in 0.362s
found 0 vulnerabilities


> protobufjs@6.9.0 postinstall /github/grpc-node/packages/proto-loader/node_modules/protobufjs
> node scripts/postinstall


> @grpc/proto-loader@0.5.5 prepare /github/grpc-node/packages/proto-loader
> npm run compile


> @grpc/proto-loader@0.5.5 compile /github/grpc-node/packages/proto-loader
> tsc -p .

../../node_modules/@types/gulp-mocha/index.d.ts:10:39 - error TS2304: Cannot find name 'MochaSetupOptions'.

10 declare function mocha(setupOptions?: MochaSetupOptions): NodeJS.ReadWriteStream;
                                         ~~~~~~~~~~~~~~~~~


Found 2 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @grpc/proto-loader@0.5.5 compile: `tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @grpc/proto-loader@0.5.5 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/net/grpc/gateway/docker/node_interop_server/Dockerfile
+++ b/net/grpc/gateway/docker/node_interop_server/Dockerfile
@@ -24,7 +24,7 @@ RUN cd packages/grpc-native-core && \
   npm link
 
 RUN cd packages/proto-loader && \
-  npm install @types/mocha && \
+  npm install @types/mocha@7.0.2&& \
   npm install --unsafe-perm
 
 WORKDIR /github/grpc-node/test

